views/settings/beacon.php

```diff
@@ -18,6 +19,7 @@
 <script type="text/javascript">window.Beacon('init', '<?php echo esc_js( $data['form_id'] ); ?>')</script>
 <script>window.Beacon("identify", <?php echo $data['identify']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>);</script>
 <script>window.Beacon("session-data", <?php echo $data['session']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>);</script>
+<script>window.Beacon("prefill", <?php echo $data['prefill']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>);</script>
 <script>window.addEventListener("hashchange", function () {
 	window.Beacon("suggest");
 }, false);</script>
```

Comment:
General question: Why do we have separate `<script></script>` elements for each of these? Why not create one `<script>` with all of the JS needed???

Reply:
Honestly I don't remember if there was a reason. I removed them and packed everything in one inline script